src/blocks/hero/test/hero.cypress.js

```diff
@@ -110,13 +110,5 @@ describe( 'Test CoBlocks Hero Block', function() {
 		} );
 
 		cy.get( '.media-toolbar-primary > .button' ).click();
-
-		cy.get( '.media-replace-flow button' ).click();
-
-		cy.get( '.components-popover__content' ).should( 'be.visible' );
-
-		cy.get( '.block-editor-media-replace-flow__media-upload-menu .components-menu-item__button' ).contains( 'Open Media Library' ).click();
-
-		cy.get( '.components-popover__content' ).should( 'not.be.visible' );
```

Comment:
Why was this removed?

Reply:
Because when executing it, the Replace flow button was not there, so I infered it had been removed from the Hero block. But in fact, maybe it was because it's a bug... Do you know if it is supposed to be there?

Reply:
Working for me. I have replaced the controls.